include/eosio/vm/vector.hpp

```diff
@@ -4,8 +4,70 @@
 
 #include <utility>
 #include <string>
+#include <vector>
 
 namespace eosio { namespace vm {
+   template <typename T>
+   class unmanaged_vector {
+      public:
+         constexpr unmanaged_vector(size_t size=0) : _data(size) {}
+         constexpr unmanaged_vector(const unmanaged_vector& v) = delete;
```

Comment:
You don't need to delete the copy constructor explicitly, when you define a move constructor.